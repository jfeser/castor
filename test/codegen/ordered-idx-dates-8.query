      AOrderedIdx(join(true, ints, select([x as y], ints)) as k,
           atuple([ascalar(k.x), ascalar(k.y)], cross), >= 3, <= 5, >= 4, <= 6)
