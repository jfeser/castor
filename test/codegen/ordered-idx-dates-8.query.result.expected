3|4|3|4
3|5|3|5
3|6|3|6
4|4|4|4
4|5|4|5
5|4|5|4
5|5|5|5
4|6|4|6
5|6|5|6
